### ISSUE TITLE
[bug] bactopia tools summary produces no html file

### ISSUE THREAD

**Issue by uloeber** [bactopia/bactopia#105]
I can produce tabular summaries, running
`/home/user/anaconda3/envs/bactopia/share/bactopia-1.3.1/tools/summary/bin/bactopia-summary.py ./ --max_contigs 3000 `

But somehow trying to use the suggested workflow, I'll get error exceptions. The summary HTML sais:
**Workflow execution completed unsuccessfully!**
The bactopia-info HTML looks good.
Any suggestions?

` bactopia tools summary --bactopia ./
N E X T F L O W  ~  version 20.01.0
Launching `/home/user/anaconda3/envs/bactopia/share/bactopia-1.3.1/tools/summary/main.nf` [clever_watson] - revision: f09dec8498
bactopia tools summary - 1.3.1
executor >  local (2)
[ba/c97efb] process > summary           [ 50%] 1 of 2, failed: 1, retries: 1
executor >  local (2)
[ba/c97efb] process > summary           [100%] 2 of 2, failed: 2, retries: 1 ✘
[-        ] process > ariba_summary     -
[-        ] process > amrfinder_summary -
[89/eb9763] NOTE: Process `summary` terminated with an error exit status (1) -- Execution is retried (1)
Error executing process > 'summary'

Caused by:
  Process `summary` terminated with an error exit status (1)

Command executed:

  bactopia-summary.py ./            --prefix bactopia         --gold_coverage 100         --gold_quality 30         --gold_read_length 95         --gold_contigs 100         --silver_coverage 50         --silver_quality 20         --silver_read_length 75         --silver_contigs 200         --min_coverage 20         --min_quality 12         --min_read_length 49         --max_contigs 500

Command exit status:
  1

Command output:
  (empty)

Command error:
  Traceback (most recent call last):
    File "/home/user/anaconda3/envs/bactopia/share/bactopia-1.3.1/tools/summary/bin/bactopia-summary.py", line 579, in <module>
      writer = csv.DictWriter(txt_fh, fieldnames=outputs[0].keys(), delimiter='\t')
  IndexError: list index out of range

Work dir:
 xxx/BACTOPIA/work/ba/c97efb99104dce3ddcf8f73ac89ba3

Tip: you can try to figure out what's wrong by changing to the process work dir and showing the script file named `.command.sh`

    Bactopia Tool 'summary' - Execution Summary
    ---------------------------
    Command Line    : nextflow run /home/user/anaconda3/envs/bactopia//share/bactopia-1.3.1/tools/summary/main.nf --bactopia ./
    Resumed         : false
    Completed At    : 2020-05-03T20:26:33.047609+02:00
    Duration        : 5.5s
    Success         : false
    Exit Code       : 1
    Error Report    : Error executing process > 'summary'
executor >  local (2)
[ba/c97efb] process > summary           [100%] 2 of 2, failed: 2, retries: 1 ✘
[-        ] process > ariba_summary     -
[-        ] process > amrfinder_summary -
[89/eb9763] NOTE: Process `summary` terminated with an error exit status (1) -- Execution is retried (1)
Error executing process > 'summary'

Caused by:
  Process `summary` terminated with an error exit status (1)

Command executed:

  bactopia-summary.py ./            --prefix bactopia         --gold_coverage 100         --gold_quality 30         --gold_read_length 95         --gold_contigs 100         --silver_coverage 50         --silver_quality 20         --silver_read_length 75         --silver_contigs 200         --min_coverage 20         --min_quality 12         --min_read_length 49         --max_contigs 500

Command exit status:
  1

Command output:
  (empty)

Command error:
  Traceback (most recent call last):
    File "/home/user/anaconda3/envs/bactopia/share/bactopia-1.3.1/tools/summary/bin/bactopia-summary.py", line 579, in <module>
      writer = csv.DictWriter(txt_fh, fieldnames=outputs[0].keys(), delimiter='\t')
  IndexError: list index out of range

Work dir:
  xxx/BACTOPIA/work/ba/c97efb99104dce3ddcf8f73ac89ba3

Tip: you can try to figure out what's wrong by changing to the process work dir and showing the script file named `.command.sh``

**Comment by rpetit3** [comment]
Looking into this. Going to see if I can replicate, I'm guessing an output file was empty (e.g. no hits) and there's no check to make sure its not empty.

I'll follow up soon.

**Comment by uloeber** [comment]
Great, thanks. The results could be parsed into summaries by 
`bactopia-summary.py ./ --max_contigs 3000 
amrfinder-summary.py ./
ariba-summary.py --include_all ./`
without any problems

**Comment by rpetit3** [comment]
Good news @uloeber I think I figured out the issue. It was due to using a relative path.

I was passing the relative path, so when Nextflow created its working directory (e.g. `work/15/3c0f2b935ac9b8d54a9bea7c60009a`) it was looking for your files at `./` in that working directory.

I've updated so that if a relative path is given it will be added to the end of the launch directory.

I'll submit a version update, but for the time being I think if you giving the absolute path should fix your issue.

Fix is here: https://github.com/bactopia/bactopia/commit/04766db0aa3fa767457d7add1cd4ed180e41a990
